File: mobile/src/main/java/com/ataulm/whatsnext/api/ApiImage.java
package com.ataulm.whatsnext.api;

import com.google.gson.annotations.SerializedName;

import java.util.List;

class ApiImage {

    @SerializedName("sizes")
    List<Size> sizes;

    static class Size {

        @SerializedName("width")
        int width;

        @SerializedName("height")
        int height;

        @SerializedName("url")
        String url;
    }
}

File: app/src/main/java/com/ataulm/whatsnext/api/ApiImage.kt
package com.ataulm.whatsnext.api

import com.google.gson.annotations.SerializedName

class ApiImage {

    @SerializedName("sizes")
    lateinit var sizes: List<Size>

    class Size {

        @SerializedName("width")
        var width: Int = 0

        @SerializedName("height")
        var height: Int = 0

        @SerializedName("url")
        lateinit var url: String
    }
}
